### ISSUE TITLE
Functionality Request: 'Hot Now' or 'News' page

### ISSUE THREAD

**Issue by dhatami** [viplavfauzdar/tommy#74]
We discussed having a wordpress type blog.
Can we create a place-holder page with either name above and embed something like this for polling?
http://www.poll-maker.com/
(I'll explain to you later why, but its something we discussed in a meeting today)

Use this as a sample poll
http://www.poll-maker.com/poll275648x964D9bEE-10

**Comment by viplavfauzdar** [comment]
So add a link in the nav bar? And a link to this website or something
similar we wanna add?

**Comment by dhatami** [comment]
Yeah a new navi bar link, and a place to display it. The results page is
interesting as well. I want to make sure the poll results look pretty cool
if possible.
